checkstyle/import-control.xml

```diff
@@ -295,6 +295,7 @@
 
   <subpackage name="tools">
     <allow pkg="org.apache.kafka.common"/>
+    <allow pkg="org.apache.kafka.server.common" />
```

Comment:
We already allow that package just below

Reply:
Fixed.